README.md

```diff
@@ -31,7 +31,7 @@ Some API methods have optional parameters that can be passed. For example:
 client := github.NewClient(nil)
 
 // list recently updated repositories for org "github"
```

Comment:
also update the comment here so that it's accurate.  Same with the one in doc.go